handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -89,17 +89,17 @@ public LoadbalanceHandler() {
 
   @Override
   public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
-    String p = Configuration.INSTANCE.getPolicy(invocation.getMicroserviceName());
+    String policy = Configuration.INSTANCE.getPolicy(invocation.getMicroserviceName());
     String strategy = Configuration.INSTANCE.getRuleStrategyName(invocation.getMicroserviceName());
-
-    if (this.policy != null && !this.policy.equals(p)
-        || (this.strategy != null && !this.strategy.equals(strategy))) {
+    boolean result = StringUtils.equals(policy,this.policy) && StringUtils.equals(strategy, this.strategy);
```

Comment:
The name of the variable `result` is not good enough to express your intention. It should be more readable.

Reply:
Okay,I hava changed to "isRuleChanged"

Reply:
@laijianbin Your new name seems not to match the comment below. Maybe it should be `boolean isRuleChanged = !StringUtils.equals(policy, this.policy) || !StringUtils.equals(strategy, this.strategy);` Then you don't need the negation in the if clause.